addons/binding/org.openhab.binding.opendaikin/pom.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+		xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.openhab.binding</groupId>
+		<artifactId>pom</artifactId>
+		<version>2.4.0-SNAPSHOT</version>
```

Comment:
Update version to `2.5.0-SNAPSHOT`. See my comment about license update. It refers to a tutorial that also includes information to bring the sources in sync with the latest from the repository. If you follow those steps and update the version here it should go well.